test/include/vectorpostprocessors/PrintBoundaryInfo.h

```diff
@@ -0,0 +1,26 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "GeneralVectorPostprocessor.h"
+
+class PrintBoundaryInfo : public GeneralVectorPostprocessor
```

Comment:
I think the name is vague and also shouldn't use the word 'Print'.

How about something like `BoundaryConnectivityInfo`? Or something of the like...